### PR TITLE
anchors: Add an accessibility hint to any links which open new windows

### PR DESCRIPTION
**Pull request by michaelwood**
Related: https://github.com/ThreeSixtyGiving/grantnav/pull/1078

### CODE REVIEW

File: src/components/02-elements/anchors/anchors.scss [hunk 1]
```diff
@@ -7,3 +7,9 @@ a {
   &:hover { color: hsla(var(--orange-dark-hsl), 1);; }
   &:active { color: hsla(var(--yellow-dark-hsl), 1);; }
 }
+
+a[target="_blank"]::after {
+  content: " (Opens in new tab)";
+  visibility: hidden;
+  voice-over: hidden;
```
Comment (codemacabre) [inline src/components/02-elements/anchors/anchors.scss:14]: I've not seen this `voice-over` CSS property before. Can you point to some documentation?

**Review by codemacabre — CHANGES_REQUESTED** [review]
This initially seems good, but it won't be visible to sighted users, nor screen-reader accessible to screen-reader users. See https://karlgroves.com/css-generated-content-is-not-content/#badusesofgeneratedcontent

Could you apply this content to the anchor elements using JavaScript so it's part of the DOM?

File: src/components/02-elements/anchors/anchors.scss (hunk 1 above)
Comment (michaelwood) [reply src/components/02-elements/anchors/anchors.scss]: Thanks yeah, copy paste error.

**Comment by michaelwood** [comment]
> This initially seems good, but it won't be visible to sighted users, nor screen-reader accessible to screen-reader users. See https://karlgroves.com/css-generated-content-is-not-content/#badusesofgeneratedcontent
> 
> Could you apply this content to the anchor elements using JavaScript so it's part of the DOM?

Hmm yeah that's a shame. Good point though.

I think might have to do this in each application with another approach.

**Pull request closed without merging**